### PR TITLE
Robotrix 2024

### PR DESCRIPTION
**Pull request by imApoorva36**
# Description

Code for conducting Robotrix 2024.

## Dependencies

List any dependencies that are required for this change.

## Type of Change

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration. (for bug fix / breaking change)
_Put an `x` in the boxes that apply_

- [ ] Test A
- [ ] Test B

---

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: corpus/corpus/settings.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = (os.getenv("ENVIRONMENT") == "DEVELOPMENT") or (
-        os.getenv("LIVECYCLE") is not None
+    os.getenv("LIVECYCLE") is not None
```
Comment (anirudhprabhakaran3) [inline corpus/corpus/settings.py:31]: Revert the change on this line

File: corpus/corpus/settings.py [hunk 2]
```diff
@@ -57,7 +57,8 @@
     "embedathon.apps.EmbedathonConfig",
     "impulse.apps.ImpulseConfig",
     "electrika.apps.ElectrikaConfig",
-    "skyward_expedition.apps.SkywardExpeditionConfig"
+    "robotrix.apps.RobotrixConfig",
+    "skyward_expedition.apps.SkywardExpeditionConfig",
```
Comment (anirudhprabhakaran3) [inline corpus/corpus/settings.py:61]: Rebase with main, this should get fixed then

**Review by anirudhprabhakaran3 — CHANGES_REQUESTED** [review]

File: corpus/corpus/settings.py (hunk 1 above)
Comment (imApoorva36) [reply corpus/corpus/settings.py:31]: done

File: corpus/corpus/settings.py (hunk 2 above)
Comment (imApoorva36) [reply corpus/corpus/settings.py]: did

**Review by anirudhprabhakaran3 — APPROVED** [review]
